source/3-tutorials/tutorial-guidelines.html.md

```diff
@@ -10,7 +10,87 @@ title: Tutorials Writing Guidelines
 
 ## Assumptions, Prerequisites & Knowledge level
 
-## Code
+## Code snippets
+
```

Comment:
I would add a recommendation about being clear using code snippets by clarifying if they are simple examples or code that must be copied to perform some actions. 
At the same time, in case you have to use placeholders in code snippets, add a note or a clarification about them. It is extremely common that users just copy the code without pay any attention to it.